runtime/zeitgeist/src/parameters.rs

```diff
@@ -80,6 +80,15 @@ parameter_types! {
     pub const TechnicalCommitteeMaxProposals: u32 = 64;
     pub const TechnicalCommitteeMotionDuration: BlockNumber = 7 * BLOCKS_PER_DAY;
 
+    // Contracts
+    pub const ContractsDeletionQueueDepth: u32 = 128;
+    pub ContractsDeletionWeightLimit: Weight = Perbill::from_percent(10) * RuntimeBlockWeights::get().max_block;
```

Comment:
Should we replace this with 10% of the weight reserved for normal dispatches?

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1012/commits/bf2035760aa69cc40512fd4e532551f203f94ece